css/main.css

```diff
@@ -0,0 +1,258 @@
+    text-decoration: none;
+}
+.sign-in{
+    display: flex;
+    position:absolute;
+    top: 5px;
+    right: 10px;
+    justify-content: space-around;
+    color: #8d9096;
+    font-size: 9px;
+    align-items: center;
+}
+
+button img{
+    height: 20px;
+    background: white;
+    border: 0px;
+    border-radius: 100%;
+    
+    
+}
+
+button p{
+    margin-left: 10px;
+    font-size: 15px;
+    color: white
+}
+
+
+
+button{
+    background: #0077c4;
+    padding: 5px 10px;
+    margin: 10px;
+    color: white;
+    border: 1.4px solid black;
+    border-radius: 4px;
+    display:flex;
+    align-items: center;
+    cursor: pointer;
+    
+}
+.create-account{
+    padding: 10px 10px;
+    border: 0px;
+    box-shadow:0px 1px 3px 0px black;
+    width: 80%;
+    width: 100%;
+    justify-content: center;
+}
+.create-account:hover{
+    background: #0084da;
+    
+}
+
+
+.title-form{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 390px;
+    margin: auto;
+    margin-top: 50px;
+    color: #8d9096;
+}
+.turbo{
+    background: red;
+    width: 20px
+}
+
+.mint{
+    width: 25px;
+}
+.quickbooks{
+    width: 25px;
+}
+.title img{
+    width: 160px;
+}
+.diferent-apps{
+    display: flex;
+    max-height: 30px;
+    align-items: center;
+}
+.apps{
+    display: flex;
+    justify-content: space-around;
+    width: 40%;
+}
+.apps img{
+
+    margin: 10px;
+    padding: 1px;
+    border-radius: 100px;
+}
+
+.form{
+    width: 390px;
+    margin: auto;
+    padding: 10px;
+    background: white;
+    margin-top: 20px;
+    border: 1px solid #c7c7c7;
+    border-radius: 2px;
+}
+form{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content:space-between;
+    height: 700px;
+    padding: 0px 20px;
+}
+
+
+.form-head{
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    text-align: center;
+    height: 20%;
+    font-size: 13px
+}
+.form-actions{
+    
+    height: 60%;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.actions{
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    height: 20%;
+    font-size: 14px;
+}
+
+.actions input{
+    padding: 10px;
+    border: none;
+    border: 1px solid #c7c7c7;
+    border-radius: 5px;
+    outline: none;
+}
+input:focus{
+    box-shadow: 0px 0px 10px 1px rgba(85,181,247,.4);
+    border: 1px solid #4faae8
+}
+.password{
+    background-image: url(../images/lock.png);
+    background-position: 99% 50%; 
+    background-repeat: no-repeat;
+    background-size:20px;
+    
+}
+
+.disclaimer{
+    width: 50%;
+    text-align: center;
+    font-size: 11px;
+    color: #8d9096;
+}
+
+.form footer{
+    position:relative;
+    padding: 10% 0%;
+    text-align: center;
+    font-size: 11px;
+    display: flex;
+
```

Comment:
Remove all unneeded whitespaces in your code.

Reply:
Solved!